sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -412,12 +412,21 @@ export class SessionReceiver {
     }
   }
 
-  private async _createMessageSessionIfDoesntExist(): Promise<void> {
+  private async _createMessageSessionIfDoesntExist(
+    onMessage?: OnMessage,
+    onError?: OnError
```

Comment:
I don't understand why this change is part of this PR which is about message settlement

Reply:
Is this related to https://github.com/Azure/azure-sdk-for-js/pull/6109#discussion_r345409362 and https://github.com/Azure/azure-sdk-for-js/pull/5693 ?

If so, please create a separate PR for handling just this problem and link that PR to the user reported issue that this solves. For posterity, it helps if bug fixes had their own commit, linked to the issue they solve rather than getting combined with another bug fix

Reply:
Reverted and filed https://github.com/Azure/azure-sdk-for-js/issues/6210